### PR TITLE
buff sleepy pens to quickly drain stamina after 10 seconds

### PR DESCRIPTION
**Pull request by silicons**
they're too unused and crappy under stamina combat, easily combatable with coffee and more or less useless unless they manage to down you first or lock you into a room.

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -304,13 +304,14 @@
...
-	metabolization_rate = 1.5 * REAGENTS_METABOLISM
+	metabolization_rate = 1 * REAGENTS_METABOLISM
 
 /datum/reagent/toxin/chloralhydratedelayed/on_mob_life(mob/living/carbon/M)
 	switch(current_cycle)
 		if(10 to 20)
 			M.confused += 1
 			M.drowsyness += 1
+			M.adjustStaminaLossBuffered(5)
 		if(20 to INFINITY)
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm:315]: knocking this down to 15 wouldnt be a bad idea.

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 2]
```diff
@@ -304,13 +304,14 @@
 	reagent_state = SOLID
 	color = "#000067" // rgb: 0, 0, 103
 	toxpwr = 0
-	metabolization_rate = 1.5 * REAGENTS_METABOLISM
+	metabolization_rate = 1 * REAGENTS_METABOLISM
 
 /datum/reagent/toxin/chloralhydratedelayed/on_mob_life(mob/living/carbon/M)
 	switch(current_cycle)
 		if(10 to 20)
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm:311]: 10-15 would be better

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 3]
```diff
@@ -304,13 +304,14 @@
...
 			M.confused += 1
 			M.drowsyness += 1
+			M.adjustStaminaLossBuffered(5)
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: 15 would make it viable

**Review by Trilbyspaceclone — APPROVED** [review]
All and all Prala pen is op weapon do to holding 30u of any chem to auto 0 delay inject, buying a toxin kit or making your own toxins with maint chemist is viable as ever and will always be the robust way to use these. Its starting mix is meant in my eyes to be a low-effect only muting and starting of sleep, well quickly loading a gun behind your back to take them out that way.
Or doing a quick pen reload mid talk or such also tends to work when I used to role antag.

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm (hunk 3 above)
Comment (deathride58) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: Affecting the stamina buffer directly is a hard no, as the stamina buffer is intended to only be affected by the user's actions and only the user's actions.

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Comment by silicons** [comment]
@deathride58 done
@Trilbyspaceclone i buffed the stam to 7.5/tick, that's 75 stam over 10 ticks and way better compounded with the tirizine. metabolization rate also makes it last longer overall. 
it is not meant to require you to go high-effort route of making your own mix. hell, i do not think chemical injections that way are balanced at all right now, mixing toxins to stack damage is bullshit and a terrible use of the goofchem system in my opinion.

**Comment by Trilbyspaceclone** [comment]
Toxin kit gives a dropper iirc and then you can mix in a 30u bottle the worst mixes ever to crit someone in 2 ticks like, no joke. So its rather good to get the bottle premade, set to 30u transfer,jab the target quick load, jab again and then drag the now dead toxin'ed airless body into a locker or throw their poor body out the airlock

**Review by deathride58 — APPROVED** [review]
This here's pretty good at the moment

**Pull request merged**